### ISSUE TITLE
twint crashes: The module usage examples from Readme don't work

### ISSUE THREAD

**Issue by DataGreed** [twintproject/twint#1019]
>Make sure you've checked the following:

- [x] Python version is 3.6;
- [x] Updated Twint with `pip3 install --user --upgrade -e git+https://github.com/twintproject/twint.git@origin/master#egg=twint`;
- [x] I have searched the issues and there are no duplicates of this issue/question/request.

### Command Ran
```
import twint

# Configure
c = twint.Config()
c.Username = "realDonaldTrump"
c.Search = "great"

# Run
twint.run.Search(c)
```

### Description of Issue
Crashes with 
```
CRITICAL:root:twint.get:User:'NoneType' object is not subscriptable
```

### Environment Details
macOS terminal, python 3.6

**Comment by AsmaZbt** [comment]
hello,
I have the same issue, I'm working on windows.
did you solve the problem, please? 

thank you

**Comment by DataGreed** [comment]
@AsmaZbt nope, I gave up on twint. Seems like it has been an issues since August when twitter changed something in their APIs.

I've wrote a custom scraping solution for my specific use-case.

**Comment by himanshudabas** [comment]
@DataGreed @AsmaZbt 
You must have installed twint using
```
pip install twint
```

try installing using thr following command, and this should work
```
pip3 install --user --upgrade -e git+https://github.com/twintproject/twint.git@origin/master#egg=twint
```

Twint hasn't been updated on pypi for a while now, that's why the first method doesn't work.

**Comment by AsmaZbt** [comment]
> @DataGreed @AsmaZbt
> You must have installed twint using
> 
> ```
> pip install twint
> ```
> 
> try installing using thr following command, and this should work
> 
> ```
> pip3 install --user --upgrade -e git+https://github.com/twintproject/twint.git@origin/master#egg=twint
> ```
> 
> Twint hasn't been updated on pypi for a while now, that's why the first method doesn't work.

hello @himanshudabas
thank you so much, the command: 

> ```
> pip3 install --user --upgrade -e git+https://github.com/twintproject/twint.git@origin/master#egg=twint
> ```

 solved the issue, twins work now for me,  but only on Jupyter Notebook and it doesn't work on Spyder, do have an idea, what can be the problem? 

best regards

**Comment by himanshudabas** [comment]
@AsmaZbt 
I haven't Personally tested twint on Spyder.
Can you share the script that you are running on Spyder along with the environment of Spyder and error that you get while running twint on Spyder.

**Comment by AsmaZbt** [comment]
> @AsmaZbt
> I haven't Personally tested twint on Spyder.
> Can you share the script that you are running on Spyder along with the environment of Spyder and error that you get while running twint on Spyder.

I fixed the issue with  : Reset Python setting 
![Capture d’écran 2020-12-03 182822](https://user-images.githubusercontent.com/32130599/101065599-a20b9780-3595-11eb-8187-0ef10394f0b5.png)


my script is very basic like this : 
c = twint.Config()
c.Username = "shugairi"
c.Limit = 1

twint.run.Search(c)

and I did not get any error message, it didn't give anything in a loop while.  

thank you so much @himanshudabas for the HELP ^_^

best regards

**Comment by nate8020** [comment]
@DataGreed what did you use for your custom scraping solution? 

I'm getting bitten by `CRITICAL:root:twint.feed:Follow:IndexError` found on issues #1127, #1101, #1077, #1011, #893 and #839...

**Comment by DataGreed** [comment]
@nate8020 I ended up using https://github.com/instaloader/instaloader - it even has iterators that you can freeze and save to resume research.

**Comment by DataGreed** [comment]
@nate8020 I just checked the xhr requests on twitter page, and created a script that automatically loads new pages using request. I don't remember all the details though as it was a one-time research, I probably re-used some iteration logic from twint.

**Comment by paulowe** [comment]
> @DataGreed what did you use for your custom scraping solution?
> 
> I'm getting bitten by `CRITICAL:root:twint.feed:Follow:IndexError` found on issues #1127, #1101, #1077, #1011, #893 and #839...

Did you figure a way around this? I get the same error on CLI when running `twint -u username --followers`

**Comment by nate8020** [comment]
@paulowe I haven't figured out a solution to this yet. I ended up using the official Twitter API as it was enough for my requirements.

**Comment by paulowe** [comment]
> @paulowe I haven't figured out a solution to this yet. I ended up using the official Twitter API as it was enough for my requirements.

Same here

**Comment by diegokenned** [comment]
> > @DataGreed o que você usou para sua solução de raspagem personalizada?
> > Estou sendo mordido por `CRITICAL:root:twint.feed:Follow:IndexError`encontrado nos problemas [# 1127](https://github.com/twintproject/twint/issues/1127) , [# 1101](https://github.com/twintproject/twint/issues/1101) , [# 1077](https://github.com/twintproject/twint/issues/1077) , [# 1011](https://github.com/twintproject/twint/issues/1011) , [# 893](https://github.com/twintproject/twint/issues/893) e [# 839](https://github.com/twintproject/twint/issues/839) ...
> 
> Você descobriu uma maneira de contornar isso? Recebo o mesmo erro na CLI ao executar`twint -u username --followers`

did you find any solution?
